src/utils/normalize-mime-parts.ts

```diff
@@ -89,6 +89,12 @@ export function normalizeMimeParts(
 				part.contentType !== 'application/pkcs7-mime' &&
 					part.contentType !== 'application/pkcs7-signature' &&
 					(acc[mode] || (acc[mode] = [])).push(processAttachment(part));
+
+				message.attributes = message.attributes || {};
+				message.attributes.isEncrypted =
+					part.contentType === 'application/pkcs7-mime';
```

Comment:
do we need check for multipart/signed also?

Reply:
Yes, we will add that when we gonna support Outlook/Thunderbird client.